terraspace/app/stacks/peer/tfvars/base.tfvars

```diff
@@ -0,0 +1,14 @@
+cluster_name = "<%= expansion(':ENV') %>-ep-peer"
+account_id   = "<%= expansion(':ACCOUNT') %>"
+region       = "<%= expansion(':REGION') %>"
+eks = {
+  name = "<%= expansion(':ENV') %>-ep-peer"
+  version = 1.21
+  eks_managed_node_groups = {
+    version        = 1.21
```

Comment:
`version` isn't allowed here (according to `variables.tf`)

Reply:
Didn't get that.. Let's talk about this

Reply:
Removed